Site/ASAB Maestro/Descriptors/elasticsearch.yaml

```diff
@@ -38,3 +38,56 @@ descriptor:
     xpack.security.transport.ssl.certificate_authorities: certs/ca.pem
     xpack.security.transport.ssl.certificate: certs/certificate.pem
     xpack.security.transport.ssl.key: certs/key.pem
+
+
+nginx:
+  https:
+    location /api/elasticsearch/:
+      - auth_request /_elasticsearch_introspect
+      - auth_request_set $batman $upstream_http_authorization
+      - proxy_set_header Authorization $batman
+
+      - auth_request_set $cookie $upstream_http_cookie
+      - proxy_set_header Cookie $cookie
+
+      - rewrite ^/api/elasticsearch/(.*) /$1 break
+      - proxy_set_header Host $host
+      - proxy_set_header Connection "Keep-Alive"
+      - proxy_set_header Proxy-Connection "Keep-Alive"
+      - proxy_pass http://elasticsearch  # upstream created by remote control technology - it leads to master
+
+    location /_elasticsearch_introspect:
+    - internal
+    - proxy_method POST
+    - proxy_set_body ""
+    - proxy_set_header X-Request-URI "request_uri"
+    - proxy_cache kibana
+    - proxy_pass http://upstream-seacat-auth-private/nginx/introspect/batman?client_id=elasticsearch
+    - proxy_cache_key $cookie_SeaCatSCI_VYNYZBRVIKLGTSQZ  # This is a cookie paired to the elasticsearch client - it contains hashed client_id - this is based on SeaCat Auth requirements.
```

Comment:
the cache key should be `$authorization`. the API calls are authorized with the token in the Authorization header, not the cookie.

Reply:
thank you :heart:

Reply:
i just checked and it should actually be `$http_authorization`. sorry for misleading you :see_no_evil: